Formula/cwlogs.rb

```diff
@@ -0,0 +1,31 @@
+class Cwlogs < Formula
+  desc "CLI tool for reading logs from Cloudwatch Logs"
+  homepage "https://github.com/segmentio/cwlogs"
+  url "https://github.com/segmentio/cwlogs/archive/v1.2.0.tar.gz"
+  sha256 "3f7b56b49c42c1de0e697fc391abad07b03434cff36b153249dd2af3107e064e"
+
+  depends_on "go" => :build
+  depends_on "govendor" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    ENV["GOOS"] = "darwin"
+    ENV["GOARCH"] = "amd64"
+    ENV["CGO_ENABLED"] = "0"
+
+    path = buildpath/"src/github.com/segmentio/cwlogs"
+    path.install Dir["{*,.git}"]
+
+    cd "src/github.com/segmentio/cwlogs" do
+      system "govendor", "sync"
+      system "go", "build", "-o", bin/"cwlogs",
+                   "-ldflags", "-X main.Version=#{version}"
+      prefix.install_metafiles
+    end
+  end
+
+  test do
+    output = shell_output("#{bin}/cwlogs help", 0)
```

Comment:
`0` is redundant; `shell_output` checks `0` by default and only needs the explicit exit code if it isn't going to be that. Any chance of a test that exercises functionality here though, rather than just asserting on the `help` output?

Reply:
I removed the explicit `0` parameter.

There really isn't any functionality we can check without valid AWS credentials.